tequilapi/endpoints/identities.go

```diff
@@ -599,6 +601,80 @@ func (ia *identitiesAPI) SavePayoutAddress(c *gin.Context) {
 	utils.WriteAsJSON(par, c.Writer)
 }
 
+// swagger:operation POST /identities/:id/migrate-hermes
+// ---
+// summary: Migrate Hermes
+// description: Migrate from old to new Hermes
+// parameters:
+// - in: path
+//   name: id
+//   description: Identity stored in keystore
+//   type: string
+//   required: true
+// responses:
+//   200:
+//     description: Successfully migrated
+//   403:
+//     schema:
+//       "$ref": "#/definitions/APIError"
+//   500:
+//     description: Internal server error
+//     schema:
+//       "$ref": "#/definitions/APIError"
+func (ia *identitiesAPI) MigrateHermes(c *gin.Context) {
+	id := c.Param("id")
+	if !ia.idm.IsUnlocked(id) {
+		c.Error(apierror.Forbidden("Identity is locked", contract.ErrCodeIDLocked))
+		return
+	}
+	err := ia.hermesMigrator.Start(id)
+	if err != nil {
+		c.Error(apierror.Internal(err.Error(), contract.ErrCodeHermesMigration))
+		log.Err(err).Msgf("could not migrate identity %s", id)
+		return
+	}
+
+	c.Status(http.StatusOK)
+}
+
+// swagger:operation GEY /identities/:id/migrate-hermes/status
```

Comment:
This endpoint makes it look like you will return a status for a migration - like is the migration going ok, but rather we're returning _if the migration is required._

Reply:
In wide sense it is. We just don't have "in progress" status, but we have kind of "to do" and "finished".